src/core/qgstracer.h

```diff
@@ -58,6 +58,22 @@ class CORE_EXPORT QgsTracer : public QObject
     //! Set extent to which graph's features will be limited (empty extent means no limit)
     void setExtent( const QgsRectangle &extent );
 
+    //! Get offset in map units that should be applied to the traced paths returned from findShortestPath().
+    //! Positive offset for right side, negative offset for left side.
+    //! \since QGIS 3.0
```

Comment:
Looks like these docstrings are not picked up by the sip files (onto which I hope we'll soon have a shiny new python api documentation built ;) )

Just guessing that the following format works better

```
/**
 * Docstring
 */
```

Reply:
Confirmed, switching over to multiline comments fixes sipdoc

https://github.com/qgis/QGIS/commit/ab9e2fcb6860912565889f80f324f219a7720de3